packages/fuels-programs/src/contract.rs

```diff
@@ -724,6 +726,7 @@ pub fn method_hash<D: Tokenizable + Parameterize + Debug, T: Account>(
     args: &[Token],
     log_decoder: LogDecoder,
     is_payable: bool,
+    encoder_config: Option<EncoderConfig>,
```

Comment:
I could not see where we use `None` so I would recommend we remove it.

```suggestion
    encoder_config: EncoderConfig,
```